src/config.rs

```diff
@@ -65,17 +77,17 @@ impl From<rcgen::RcgenError> for ConfigError {
     }
 }
 
-impl From<quinn::ParseError> for ConfigError {
-    fn from(error: quinn::ParseError) -> Self {
-        Self::CertificateGeneration(CertificateGenerationError(error.into()))
-    }
-}
+// impl From<quinn::ParseError> for ConfigError {
```

Comment:
Was this intentional?

Reply:
ah, needs removed. 

quinn config is now infallible